src/component/Animatable.lua

```diff
@@ -0,0 +1,16 @@
+--Animatable.lua
```

Comment:
I like the idea of making the Renderable switch behaviour based on the asset: we'll go with that I think. Later we can perhaps play with the idea of a Renderable:playOnce(animation) functionality.